doc/config.rst

```diff
@@ -200,6 +200,10 @@ repository.
 ``package-name-version.tar.gz`` files in the directory, and will use optional
 corresponding ``package-name-version.cabal`` files as new revisions.
 
+.. note::
+   On Windows systems, the path has to be prefixed by ``//./`` as in
```

Comment:
The changelog says “can”, but docs says “has to”. These are different things, right?

Reply:
I meant to say that now they _can_ use local+noindex repos, previously they couldn't. And if they want to use it they _have to_ use that syntax.

Isn't the wording right?

Reply:
Yes, the "if" part that you say here (`if they want to use it they have to use that syntax`) is not in the docs. The docs just say "has". Could you add the "if" part?

Reply:
I don't understand this change. `local+noindex` repositories are not mandatory, this whole section is for the user to read if they want to use them. I think it is implied by the docs as a whole. If you don't want to use them you don't even care about this section of the docs.

For example https://cabal.readthedocs.io/en/stable/cabal-project-description-file.html#cfg-field-packages doesn't say "If you want packages in your project then you have to fill in this field". The whole point of describing the field is for the user to use if they want to.

Reply:
I thought when you said “ if they want to use it”, by “it” you mean this specific form of the path, not no-index repositories in general. Is this particular form the only way to have a working no-index repo on Windows? If the answer is yes, then the “can” in the changelog is incorrect send should be “must”.

Reply:
https://github.com/haskell/cabal/compare/908b17edb18038325a6d776ef6b653dac75613af..a48e96ab5b336add3485cf38e989fb4f83d1a76a

What about this.

Indeed this is the only way to have working no-index repos on Windows in a sane way. One can use `file+noindex:///some/path` and that will look in `<CURRENT_DRIVE>:/some/path` which is very fragile and therefore IMHO should not be used nor documented.

Reply:
> What about this

Terrific, thank you! Resolving…